### DISCUSSION TITLE
"Burkina Faso" not detected

### DISCUSSION THREAD

**Discussion by joaobarcia** [explosion/spaCy#8140] [Help: Other Questions] [answered]
Spacy does not detect `Burkina Faso`. 

I work for an NGO that does data analysis on violent incidents regarding the humanitarian and education sectors. We are using Spacy to automate our data entry pipeline. It works amazing most of the times, but several of our events are in Burkina Faso and Spacy does not seem to be able to detect this country in specific. I think all other countries are detected.

## How to reproduce the behaviour

```
import spacy

nlp = spacy.load("en_core_web_sm")
doc = nlp("Burkina-Faso, 13 February 2020: In Moussakuy village, Sanaba department, Boucle du Mouhoun region, an unnamed school was attacked by a group of ten heavily armed men who burned electoral material and seized food from the canteen")
print(doc.ents)
```
Returns 
```
(13 February 2020, Moussakuy, Sanaba, Boucle, Mouhoun, ten)
```

## Your Environment
- **spaCy version:** 3.0.6
- **Platform:** Linux-5.11.0-7614-generic-x86_64-with-debian-bullseye-sid
- **Python version:** 3.7.9
- **Pipelines:** en_core_web_sm (3.0.0)

**Comment by polm** [comment, marked as answer]
This is deeply unfortunate but not a bug (error in code) or something we can fix directly. 

The models are statistical and make mistakes. Given the training data, it would be unsurprising if Burkina Faso wasn't mentioned that frequently, and it's also two separate unusual words, which is unusual for a country name, so it's kind of hard for the model to recognize. You can read more about errors in the models in #3052.

The good news is it's detected in the medium and large models. 

```
import spacy
nlp = spacy.load("en_core_web_md")
text = "I went to Burkina Faso for vacation."
for ent in nlp(text).ents:
    print(ent.label_, ent, sep="\t")
``` 

Because this is not a bug I'm going to move it to a discussion, please feel free to follow up there if you want.

**Reply by polm** [reply]
As an additional note, when you have specific entities you need for your application that aren't being picked up by a model for any reason, that's a good time to use the [EntityRuler](https://spacy.io/usage/rule-based-matching#entityruler).

**Reply by joaobarcia** [reply]
Ah, thanks for the quick reply and providing already two possible solutions! :) I'll try it out this week. I'm not sure if discussions are supposed to get "closed" but, if they do, feel free to close the issue/discussion.

**Reply by joaobarcia** [reply]
Just noticed that the original issue was closed, so that's probably it. Thank again

**Reply by polm** [reply]
Ah, yeah - it's a little confusing but when an issue is migrated to a discussion, the issue is closed and locked, and then the comments are copied to the discussion. Sorry for the confusion.
